packages/utils/feed/markdown.ts

```diff
@@ -0,0 +1,118 @@
+import markdownit from "markdown-it";
+import { full as emoji } from "markdown-it-emoji/dist/index.cjs";
+import { MixedStyleRecord } from "react-native-render-html";
+
+import {
+  neutral17,
+  neutralA3,
+  neutralFF,
+  primaryColor,
+} from "@/utils/style/colors";
+
+export type ContentMode = "EDITION" | "BOTH" | "PREVIEW";
+
+export const markdownTagStyles: MixedStyleRecord = {
```

Comment:
nested list have weird spacing, for example on https://deploy-preview-1451--teritori-dapp.netlify.app/feed/post/testori-d459eafc-dc3a-4e10-be9b-ab286b6485fd

<img width="200" alt="Screenshot 2024-12-17 at 09 26 02" src="https://github.com/user-attachments/assets/593f8613-67e8-4214-919b-13b33b5f3813" />


for example on github:

- foo
  - bar
  - baz
- fi
  - fa
  - fo

Reply:
https://github.com/TERITORI/teritori-dapp/pull/1451/commits/9c42f452b7854b855dffac21b1c61a3212b74a0e#diff-a4fc1763569698c2bfa6b5a6128fc846846a372cf52b9d00faf7fc560eba5afdR32-R40